contracts/libraries/Vault.sol

```diff
@@ -60,6 +60,8 @@ library Vault {
         uint128 totalPending;
         // Amount locked for scheduled withdrawals;
         uint128 queuedWithdrawShares;
+        // Amount locked for scheduled withdrawals last week;
+        uint128 lastQueuedWithdrawAmount;
```

Comment:
We can't append `lastQueuedWithdrawAmount` into the `VaultState` because it will break the storage layout. So we need to append it into RibbonThetaVaultStorageV2 as a variable

```solidity
contract RibbonThetaVaultStorageV2 {
  uint256 lastQueuedWithdrawAmount;
}
```

Reply:
addressed this